Add sciserver information to README (#134)
NASA-NAVO/navo-workshop: README.md

being left unattended for several minutes.

[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/NASA-NAVO/notebooks/main?urlpath=lab)

## Running on Sciserver

> The workshop notebooks can also be run on [Sciserver](https://sciserver.org/), which offer an online jypyter platform. To use it for the workshop, follow these steps:

Reviewer: ```suggestion
The workshop notebooks can also be run on [Sciserver](https://sciserver.org/), which offers an online Jupyter platform. To use it for the workshop, follow these steps:
```

> - Create an account on sciserver, if you do not already have one.

Reviewer: ```suggestion
- Create an account on Sciserver, if you do not already have one.
```

- Once logged in, select the *Compute* app.

> - Create a container, give it name, select the *'Sciserver Essentials 2.0'* image, and click *Create*. Click on the container name to launch the jupyterlab interface.

Reviewer: ```suggestion
- Create a container, give it name, select the *'Sciserver Essentials 2.0'* image, and click *Create*. Click on the container name to launch the JupyterLab interface.
```
